packages/status-page/src/utils/getNumProvers.ts

```diff
@@ -11,8 +11,20 @@ export type UniqueProverResponse = {
 export const getNumProvers = async (
   eventIndexerApiUrl: string
 ): Promise<UniqueProverResponse> => {
-  const resp = await axios.get<UniqueProverResponse>(
+  const uniqueProversResp = await axios.get<UniqueProverResponse>(
     `${eventIndexerApiUrl}/uniqueProvers`
   );
-  return resp.data;
+  if (uniqueProversResp.data) {
+    uniqueProversResp.data.provers.sort((a, b) => b.count - a.count);
+    // Filter out the oracle prover address since it doesn't submit the actual zk proof
+    const index = uniqueProversResp.data.provers.findIndex(
+      (uniqueProver) =>
+        uniqueProver.address === "0x0000000000000000000000000000000000000000"
```

Comment:
you can use `ethers.constants.AddressZero` here but i think thats just a nitpick